src/compiler/rust.rs

```diff
@@ -227,7 +224,7 @@ where
         .envs(ref_env(env_vars))
         .current_dir(cwd);
     trace!("[{}]: get dep-info: {:?}", crate_name, cmd);
-    let dep_info = run_input_output(cmd, None).await?;
+    let _dep_info = run_input_output(cmd, None).await?;
```

Comment:
This must be understood, why are we calling `run_input_output` if we don't care about the result. There must be comment for explaining the code.

Reply:
Added comment clarifying: 13d24c7